.drone.jsonnet

```diff
@@ -354,6 +354,7 @@ local integration_qemu = Step("e2e-qemu", privileged=true, depends_on=[load_arti
 local integration_provision_tests_prepare = Step("provision-tests-prepare", privileged=true, depends_on=[load_artifacts]);
 local integration_provision_tests_track_0 = Step("provision-tests-track-0", privileged=true, depends_on=[integration_provision_tests_prepare], environment={"REGISTRY": local_registry});
 local integration_provision_tests_track_1 = Step("provision-tests-track-1", privileged=true, depends_on=[integration_provision_tests_prepare], environment={"REGISTRY": local_registry});
+local integration_provision_tests_track_2 = Step("provision-tests-track-2", privileged=true, depends_on=[integration_provision_tests_prepare], environment={"REGISTRY": local_registry, "WITH_DISK_ENCRYPTION": "true"});
```

Comment:
This environment variable has no effect on integration tests, all config should be done in the test spec (in the code)